src/Pages/HomePage/index.jsx

```diff
@@ -0,0 +1,12 @@
+import InboxSummary from "../../Components/InboxSummary"
```

Comment:
This HomePage component doesn't really do very much - it's just adding an extra layer into your component tree. 

I would suggest removing it, and plugging the InboxSummary directly into the route - at which point you would want to move InboxSummary into the pages folder.